README.md

```diff
@@ -20,12 +22,15 @@
 
 `Frameworks`
 
-- SASS
 - Webpack
 
 ## 📡 Live Demo
 
-<!-- [Link to Live Demo](https://leaderboard-webpack.netlify.app/) -->
+[Link to Live Demo](https://yaseerokino.github.io/javascript-capstone/)
+
+## 🎥 Video Link
+
+[Link to Project Video Description](https://www.loom.com/share/2f2b19014b6c42e69442095a2030a801)
```

Comment:
- [ ] Great job on the video I loved it!! However, a hard requirement that ought to be adhered to is that everyone that participated in building the project must appear and talk about it. Currently, I can only see Lilian and Kossi but I cannot see the third member. For this reason, you have to remake another video.